### DISCUSSION TITLE
Is there a public Power Platform CLI repo?

### DISCUSSION THREAD

**Discussion by martin-dreyer-teck** [microsoft/powerplatform-vscode#341] [Q&A] [answered]
I'm looking for the Power Platform CLI repo that corresponds to [this product](https://learn.microsoft.com/en-us/power-platform/developer/cli/introduction).

This powerplatform-vscode repo is the closest I've found and I'm wondering whether anyone here can point me to the repo where the PAC CLI itself is being developed.

**Comment by devkeydet** [comment, marked as answer]
There is no public repo for the Power Platform CLI.  But the same overall team that builds the vscode extension builds the CLI.
